feign-hystrix-opentracing/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>io.github.openfeign.opentracing</groupId>
     <artifactId>feign-opentracing-parent</artifactId>
-    <version>0.0.5-SNAPSHOT</version>
+    <version>0.0.6-SNAPSHOT</version>
```

Comment:
There shouldn't be this change. The PR wasn't opened against HEAD. Rebase didn't help. I will open new PR